content/Software Support/Installation/Uninstall-Arduino-IDE.md

```diff
@@ -10,7 +10,7 @@ Learn how to uninstall Arduino IDE.
 ## Uninstalling the application
 
 * **Windows:**
-  * In Windows 10: Open the start menu and find the Arduino application. Right-click on it and select **Uninstall**.
+  * In Windows 10/11: Open the start menu and find the Arduino application. Right-click on it and select **Uninstall**.
   * Alternatively, open Explorer and run `C:\Program Files\Arduino IDE\Uninstall Arduino IDE.exe`. Or if you're using Arduino IDE 1.x, `C:\Program Files (x86)\Arduino\uninstall.exe`.
```

Comment:
This information isn't complete as is (because of the **All users** / **Only me** option). In this suggestion, instead of expanding on this point with additional information, I've linked a relevant artice.

```suggestion
  * Alternatively, [open the Arduino IDE installation folder in Explorer](https://support.arduino.cc/hc/en-us/articles/4412943340178-Open-the-Arduino-IDE-installation-folder#windows) and run the `Uninstall Arduino IDE.exe` (IDE 2) / `uninstall.exe` (IDE 1) executable.
```

Reply:
Good catch 👍